moduleframework/avocado_testers/avocado_test.py

```diff
@@ -243,15 +243,15 @@ def getArch(self):
         """
         return self.backend.getArch()
 
-    def getModuleDependencies(self):
+    def dependency_list(self):
```

Comment:
please let there backward compatibility for every function. Probably as @TomasTomecek with using warning.warn(...) that it will be removed in future, to give chance to users to change behaviour.